ST/30_api_model_rest_basics/Api_Model_and_Rest_Basics_README.md

```diff
@@ -0,0 +1,391 @@
+
+<p style="font-size: 45px; text-align: center;">
+<strong> API Model. Rest Basics </strong>
+</p>
+
+<div style="text-align: center;">
+    <img width="25%" src="images/api-anim.gif">
+</div>
+
+
+***
+
+## Contents
+
+1. [Brief Overview of API Testing and Importance of Automated API Testing](#introduction-to-api-and-importance)
+2. [Understanding the concept of API modeling](#api-modeling-concept)
+3. [Types of API models](#types-of-api-model)
+4. [Understanding Rest and its Importance in Modern Web Development](#introduction-to-rest-and-importance)
+5. [Understanding the six constraints of REST architecture](#rest-arhitecture-constraints)
+6. [RESTful API Design Best Practices](#api-design-best-practice)
+7. [Exploring Tools for API Development and Management](#tools-api)
+8. [Resources](#resources)
+
+***
+
+### Brief Overview of API Testing and Importance of Automated API Testing
+
+API (Application Programming Interface)is a set of rules and tools that
+allows different software applications to communicate with each other. APIs define the methods and data formats that
+applications can use to request and exchange information. They enable developers to access the functionality of a
+particular software component or service without exposing its internal details.
+
+Simply said, an API is a collection of guidelines and procedures which enable communication between various software
+applications. The significance of API is that it serves as a middleman, allowing developers to create software that
+makes use of the functionality and data offered by another application or service.
+
+Let's take the scenario where you use a travel booking app to look for the best hotel offers in San Francisco. To
+receive information about hotel rates from different hotel booking websites, this app may use an API. You may compare
+prices and easily reserve the best offer thanks to the API's smooth data processing and return to the app.
+***
+<h3 style="color:AliceBlue"><strong>Why is API Testing Essential?</strong></h3>
+
+1. <p style="color:DodgerBlue">Improved Functionality </p>
+   <div style="text-align:center">
+    <img src="images/improved-functionality.png" width="15%" alt="Improved Functionality">
```

Comment:
Do you think that images under these points are necessary? From my point of view, they make reading slightly difficult.

Reply:
In my opinion this removes the broodiness from the reading of plain text and add some colors that make the presentation lively. I would keep them.

Reply:
Then let's keep them. Closing this one.